tensorrt_llm/_torch/modules/fused_moe/moe_backend.py

```
@@ -0,0 +1,793 @@
+        tp_rank = module.tp_rank
+        ep_size = module.ep_size
+        ep_rank = module.ep_rank
+        cluster_size = module.cluster_size
+        cluster_rank = module.cluster_rank
+        enable_alltoall = module.enable_alltoall
+        getattr(module, 'tune_max_num_tokens', 8192)
+        swiglu_alpha = module.swiglu_alpha
+        swiglu_beta = module.swiglu_beta
+        swiglu_limit = module.swiglu_limit
+        # Extract quantization flags from module properties
+        use_deepseek_fp8_block_scale = getattr(module,
```

Ruff (0.12.2)

Local variable `use_deepseek_fp8_block_scale` is assigned to but never used

Remove assignment to unused variable `use_deepseek_fp8_block_scale`

(F841)